pkg/api/grids/grids_test.go

```diff
@@ -0,0 +1,142 @@
+// Copyright (C) 2020, Oracle and/or its affiliates.
+// Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+package grids
+
+import (
+	"encoding/json"
+	"github.com/gorilla/mux"
+	"github.com/stretchr/testify/assert"
+	"github.com/verrazzano/verrazzano-operator/pkg/controller"
+	"github.com/verrazzano/verrazzano-operator/pkg/testutil"
+	"github.com/verrazzano/verrazzano-operator/pkg/testutilcontroller"
+	"github.com/verrazzano/verrazzano-operator/pkg/types"
+	"k8s.io/client-go/kubernetes"
+	k8sfake "k8s.io/client-go/kubernetes/fake"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+)
+
+var modelBindingPairs = map[string]*types.ModelBindingPair{
+	"test-pair-1": testutil.GetModelBindingPair(),
+}
+
+// TestReturnAllGrids tests that all Grids are returned
+// GIVEN that Grids are configured
+// WHEN I call GetAllGrids
```

Comment:
Should say?
```
WHEN I call ReturnAllGrids
```

Reply:
ideally it should describe what it does, not just name the function... but not a big deal right now